docs/sources/vendor/Dell/avamar.md

```diff
@@ -0,0 +1,25 @@
+# Dell Avamar
+
+## Key facts
+
+* Requires vendor product by source configuration
```

Comment:
@ikheifets-splunk can you change it to `MSG Format based filter`?

Reply:
Sure, fixed. Very thanks I forgot to change it :)